src/libs/ck-libs/tcharm/tcharm_impl.h

```diff
@@ -103,6 +103,8 @@ class TCharm: public CBase_TCharm
 	TCharmInitMsg *initMsg; //Thread initialization data
 	double timeOffset; //Value to add to CkWallTimer to get my clock
 
+  CkCallback resumeCallback;
```

Comment:
Maybe add an inline comment describing the use of this variable

Reply:
Done